ci/groups.yml

```diff
@@ -2,7 +2,6 @@ groups:
   - name: master
     jobs:
       - development
-      - qa
```

Comment:
This should be kept and commented out too if you are commenting out the job code